### ISSUE TITLE
Emacs daemon startup hangs with (require 'eclim)

### ISSUE THREAD

**Issue by winny-** [senny/emacs-eclim#285]
Hi, my emacs daemon appears to hang forever on startup when eclim is required. When I start up emacs (no daemon), it starts up as expected. It is possible I may have misconfigured eclim, but this behavior is strange none-the-less.

I'm reluctant to share my init.el, but if asked, I'll clean it up, test again (to be sure) and share.

**Comment by nloyola** [comment]
Development on this project has moved to [emacs-eclim/emacs-eclim](https://github.com/emacs-eclim/emacs-eclim). Are you using the version published by that repository?

**Comment by winny-** [comment]
Oops! My mistake! Thanks for commenting on this ticket on the old repo.

I'm running version 20170328.1807. Is this up to date, I apologize, I'm uncertain how to verify. 

Unrelated suggestion: I suggest adding a CONTRIBUTING.md in this old repo saying the project has moved (in addition to what is already done with the repo README and description). This way users have yet another chance to realize this is the old repository.

**Comment by nloyola** [comment]
@winny- yes 20170328.1807 is the latest version from [emacs-eclim/emacs-eclim](https://github.com/emacs-eclim/emacs-eclim). 

Can you please submit this issue to that repository?
